gunagala/camera.py

```diff
@@ -39,6 +39,7 @@ class Camera:
     Resolution : astropy.units.Quantity
         Two element Quantity containing the number of pixels across
         the image sensor in both horizontal & vertical directions.
```

Comment:
Replace "horizontal & vertical direction" with "vertical & horizontal directions" just to make this really clear.